### PR TITLE
Add explanations for endocardial specificity and human snRNAseq analysis

### PR DESCRIPTION
**Pull request by FloWuenne**
This PR adds more documentation to the processing of human snRNA-seq data and statistical tests performed between Donor and AMI samples (R1-4).

### CODE REVIEW

File: analysis/figures.Figure5.Rmd [hunk 1]
```diff
@@ -416,20 +420,25 @@ sub_human_cite_pb_de <- FindMarkers(object = sub_human_cite_pb,
...
 ```
 
+Finally, let's plot a violin plot for all cells across all samples in the endocardial cluster. First, we will plot a version with only two violins, one for each group.
 
 ```{r}
 Idents(sub_human_cite) <- sub_human_cite$disease_group
 vwf_vlnpot <- SCpubr::do_ViolinPlot(sample = sub_human_cite, 
                       features = "VWF",
-                      group.by = "disease_group",
+                      group.by = "disease_gro up",
```
Comment (kbestak) [inline analysis/figures.Figure5.Rmd:441]: Is the space in "disease_gro up" on purpose?

File: analysis/proteomics.scRNAseq_comparison.Rmd [hunk 2]
```diff
@@ -131,7 +131,7 @@ endocard_de <- endocard_de %>%
 
 ## Get endocardial specific genes
 
-To get an estimate of which genes are most specifically expressed in endocardialc ells in the snRNA-seq data, we will use the FindMarkers function from Seurat to get log-fold changes and p-values for the comparison of endocardial cells at day 0 to all other cells.
+To get an estimate of which genes are most specifically expressed in endocardialc ells in the snRNA-seq data, we will use the FindMarkers function from Seurat to get log-fold changes and p-values for the comparison of endocardial cells from Sham controls to all other cells.
```
Comment (kbestak) [inline analysis/proteomics.scRNAseq_comparison.Rmd:134]:
```suggestion
To get an estimate of which genes are most specifically expressed in endocardial cells in the snRNA-seq data, we will use the FindMarkers function from Seurat to get log-fold changes and p-values for the comparison of endocardial cells from Sham controls to all other cells.
```

**Review by kbestak — COMMENTED** [review]
To me this looks good, I found potentially two typos and left comments.

File: analysis/figures.Figure5.Rmd (hunk 1 above)
Comment (FloWuenne) [reply analysis/figures.Figure5.Rmd]: No that was obviously a type 😅 !

**Pull request merged**